Week-3/session-09/passwordStrength.js

```diff
@@ -0,0 +1,17 @@
+function isStrongPassword(password){
+	if( (password.length < 8) || password.includes('password') || !containsUppercase(password) ){
+		return false
+	}
+	return true
+}
+
+function containsUppercase(str) {
```

Comment:
if (!/[A-Z]/.test(password)) {
        return false;
    }
u can use regex to handle this

Reply:
done